pkg/ui/src/views/shared/components/dropdown/index.tsx

```diff
@@ -33,6 +34,26 @@ interface DropdownOwnProps {
  * Dropdown component that uses the URL query string for state.
  */
 export default class Dropdown extends React.Component<DropdownOwnProps, {}> {
+  dropdownRef: React.RefObject<HTMLDivElement> = React.createRef();
+  titleRef: React.RefObject<HTMLDivElement> = React.createRef();
+  selectRef: React.RefObject<ReactSelectClass> = React.createRef();
+
+  constructor(props: DropdownOwnProps) {
+    super(props);
+
+    this.triggerSelectClick = this.triggerSelectClick.bind(this);
+  }
+
+  triggerSelectClick(e: any) {
+    const dropdownNode: any = this.dropdownRef.current as Node;
+    const titleNode: any = this.titleRef.current as Node;
+    const selectNode: any = this.selectRef.current;
+
+    if (e.target === dropdownNode || e.target === titleNode || e.target.className.indexOf("dropdown__select") > -1) {
```

Comment:
What might be another way of achieving the same effect (i.e. targeting `div.dropdown__select` rendered by the `Select` component)?  

The issue was that I couldn't check directly against a ref (as with `dropdownNode` and `titleNode`) because the ref would be placed on a `ReactSelectClass` and couldn't be cast as a `Node` to permit the equivalency check used in the former two cases.